base/server/share/conf/pki.policy

```diff
@@ -13,21 +13,11 @@
 grant codeBase "file:${catalina.home}/bin/tomcat-juli.jar" {
         permission java.lang.RuntimePermission "accessClassInPackage.sun.util.logging.resources";
 
-        // Allow Tomcat JULI to read subsystem logging configuration.
-        permission java.io.FilePermission "/usr/share/pki/server/webapps/pki/WEB-INF/classes/logging.properties", "read";
-        permission java.io.FilePermission "/usr/share/pki/ca/webapps/ca/WEB-INF/classes/logging.properties", "read";
-        permission java.io.FilePermission "/usr/share/pki/kra/webapps/kra/WEB-INF/classes/logging.properties", "read";
-        permission java.io.FilePermission "/usr/share/pki/ocsp/webapps/ocsp/WEB-INF/classes/logging.properties", "read";
-        permission java.io.FilePermission "/usr/share/pki/tks/webapps/tks/WEB-INF/classes/logging.properties", "read";
-        permission java.io.FilePermission "/usr/share/pki/tps/webapps/tps/WEB-INF/classes/logging.properties", "read";
+        // Allow Tomcat JULI to read subsystem logging.properties files.
+        permission java.io.FilePermission "/usr/share/pki/-", "read";
```

Comment:
I'm not an expert here. But won't this give the subsystems access to far more than "just" `logging.properties` files?

Reply:
Yes, but I don't see a problem with that. Those are shared files owned by the package and they are supposed to be read-only.

Reply:
Hmmm, I guess that's fair. We generally assume that any local process should be able to read those files.

Just that the comment seems misleading if we're granting access to everything. :)